cli/admin/src/test/java/io/pravega/cli/admin/utils/TestUtils.java

```diff
@@ -222,6 +222,29 @@ public static void createScopeStream(Controller controller, String scopeName, St
         log.info("Create stream status {}", createStreamStatus);
     }
 
+    /**
```

Comment:
How is this change related to this PR?

Reply:
Added a method to delete a scope and stream to help during testing.

Reply:
Ok